lib/ansible/modules/network/junos/junos_lldp_interfaces.py

```diff
@@ -0,0 +1,224 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+#############################################
+#                WARNING                    #
+#############################################
+#
+# This file is auto generated by the resource
+#   module builder playbook.
+#
+# Do not edit this file manually.
+#
+# Changes to this file will be over written
+#   by the resource module builder.
+#
+# Changes should be made in the model used to
+#   generate this file or in the resource module
+#   builder template.
+#
+#############################################
+
+"""
+The module file for junos_lldp_interfaces
+"""
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'network'
+}
+
+DOCUMENTATION = """
+---
+module: junos_lldp_interfaces
+version_added: 2.9
+short_description: Manage link layer discovery protocol (LLDP) attributes of interfaces on Juniper JUNOS devices
+description:
+  - This module manages link layer discovery protocol (LLDP) attributes of interfaces on Juniper JUNOS devices.
+author: Ganesh Nalawade (@ganeshrn)
+options:
+  config:
+    description: The list of link layer discovery protocol interface attribute configurations
+    type: list
+    elements: dict
```

Comment:
Check the error on shippable here: ERROR: lib/ansible/modules/network/junos/junos_lldp_interfaces.py:0:0: E305 DOCUMENTATION.options.config.elements: extra keys not allowed @ data['options']['config']['elements']. Got 'dict' (75%)

Reply:
This CI failure is fixed by PR https://github.com/ansible/ansible/pull/59244.
Since it is merged sanity test is passing now for this PR

Reply:
As no other changes are required, I will merge this PR